core/src/avm1/globals/function.rs

```diff
@@ -86,7 +86,13 @@ pub fn apply<'gc>(
     let mut child_args = Vec::with_capacity(length);
     while child_args.len() < length {
         let args = args_object.coerce_to_object(activation);
-        let next_arg = args.get(&format!("{}", child_args.len()), activation)?;
+        // TODO(moulins): avoid extra alloc?
```

Comment:
I also don't think this is avoidable, since `format!` itself allocates.

Reply:
Same as above.